config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,24 +3,24 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "08 January 2023",
+                "cves": "CVE-2022-42898",
```

Comment:
`"cves": "CVE-2022-42898",` ----->
`"cves": "CVE-2022-42898,CVE-2022-41717,CVE-2022-41720",`

and 

`"fixes": [
                    "Fixes CVE-2022-42898,CVE-2022-41717,CVE-2022-41720 and PSIRT PVR0406812,PVR0410879,PVR0410611"
                 ]`
-----> `"fixes": []`